config/webpack.config.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 'use strict';
+
+const path = require('path');
+const webpack = require('webpack');
+const ChunkWebpack = webpack.optimize.CommonsChunkPlugin;
+
+const rootDir = path.resolve(__dirname, '..');
+
+const FOAM_DIR = `${__dirname}/../node_modules/foam2`;
+const execSync = require('child_process').execSync;
+execSync(`node ${FOAM_DIR}/tools/build.js  web,gcloud`);
+
+module.exports = {
```

Comment:
Let's not use webpack for this. We can `execSync('mv ${FOAM_DIR}/foam-bin.js ${rootDir}/static/bundle/foam.bundle.js');`.

Also, let's make our string constants consistent (and maybe also create a "bundle dir" constant for copying FOAM?). I prefer consistently `NAMED_LIKE_THIS` for string constants, but `namedLikeThis` is okay too (since `const` tells us they are const).

This should eliminate the need for webpack-related stuffs as devDependencies, right?